neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -0,0 +1,185 @@
+            [OpCode.PUSHDATA1] = 180,
+            [OpCode.PUSHDATA2] = 13000,
+            [OpCode.PUSHDATA4] = 110000,
+            [OpCode.PUSHM1] = 30,
+            [OpCode.PUSH1] = 30,
+            [OpCode.PUSH2] = 30,
+            [OpCode.PUSH3] = 30,
+            [OpCode.PUSH4] = 30,
+            [OpCode.PUSH5] = 30,
+            [OpCode.PUSH6] = 30,
+            [OpCode.PUSH7] = 30,
+            [OpCode.PUSH8] = 30,
+            [OpCode.PUSH9] = 30,
+            [OpCode.PUSH10] = 30,
+            [OpCode.PUSH11] = 30,
+            [OpCode.PUSH12] = 30,
+            [OpCode.PUSH13] = 30,
+            [OpCode.PUSH14] = 30,
+            [OpCode.PUSH15] = 30,
+            [OpCode.PUSH16] = 30,
+            [OpCode.NOP] = 30,
+            [OpCode.JMP] = 70,
+            [OpCode.JMPIF] = 70,
+            [OpCode.JMPIFNOT] = 70,
+            [OpCode.CALL] = 22000,
+            [OpCode.RET] = 40,
+            [OpCode.SYSCALL] = 0,
+            [OpCode.DUPFROMALTSTACKBOTTOM] = 60,
+            [OpCode.DUPFROMALTSTACK] = 60,
+            [OpCode.TOALTSTACK] = 60,
+            [OpCode.FROMALTSTACK] = 60,
+            [OpCode.XDROP] = 400,
+            [OpCode.XSWAP] = 60,
+            [OpCode.XTUCK] = 400,
+            [OpCode.DEPTH] = 60,
+            [OpCode.DROP] = 60,
+            [OpCode.DUP] = 60,
+            [OpCode.NIP] = 60,
+            [OpCode.OVER] = 60,
+            [OpCode.PICK] = 60,
+            [OpCode.ROLL] = 400,
+            [OpCode.ROT] = 60,
+            [OpCode.SWAP] = 60,
+            [OpCode.TUCK] = 60,
+            [OpCode.CAT] = 80000,
+            [OpCode.SUBSTR] = 80000,
+            [OpCode.LEFT] = 80000,
+            [OpCode.RIGHT] = 80000,
+            [OpCode.SIZE] = 60,
+            [OpCode.INVERT] = 100,
+            [OpCode.AND] = 200,
+            [OpCode.OR] = 200,
+            [OpCode.XOR] = 200,
+            [OpCode.EQUAL] = 200,
+            [OpCode.INC] = 100,
+            [OpCode.DEC] = 100,
+            [OpCode.SIGN] = 100,
+            [OpCode.NEGATE] = 100,
+            [OpCode.ABS] = 100,
+            [OpCode.NOT] = 100,
+            [OpCode.NZ] = 100,
+            [OpCode.ADD] = 200,
+            [OpCode.SUB] = 200,
+            [OpCode.MUL] = 300,
+            [OpCode.DIV] = 300,
+            [OpCode.MOD] = 300,
+            [OpCode.SHL] = 300,
+            [OpCode.SHR] = 300,
+            [OpCode.BOOLAND] = 200,
+            [OpCode.BOOLOR] = 200,
+            [OpCode.NUMEQUAL] = 200,
+            [OpCode.NUMNOTEQUAL] = 200,
+            [OpCode.LT] = 200,
+            [OpCode.GT] = 200,
+            [OpCode.LTE] = 200,
+            [OpCode.GTE] = 200,
+            [OpCode.MIN] = 200,
+            [OpCode.MAX] = 200,
+            [OpCode.WITHIN] = 200,
+            [OpCode.SHA1] = 300000,
+            [OpCode.SHA256] = 1000000,
+            [OpCode.ARRAYSIZE] = 150,
+            [OpCode.PACK] = 7000,
+            [OpCode.UNPACK] = 7000,
+            [OpCode.PICKITEM] = 270000,
```

Comment:
Maybe we should optimize this Opcode, is very used

Reply:
After optimization, we can modify this price. But optimizing this opcode is not something that this PR needs to do.